src/udev/udevadm-control.c

```diff
@@ -161,7 +161,7 @@ int control_main(int argc, char *argv[], void *userdata) {
 
                         r = safe_atou(optarg, &i);
                         if (r < 0)
-                                return log_error_errno(r, "Failed to parse maximum number of events '%s': %m", optarg);
+                                return log_error_errno(r, "Failed to parse maximum number of children '%s': %m", optarg);
```

Comment:
i cherry picked this one directly into git main, since it's independent of the rest of the PR i guess.